<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PRwindow</class>
 <widget class="QDialog" name="PRwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ButtonAddPR">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add PR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonShowPR">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show PR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAddPRweight">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add PR for weights</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAddPRrunning">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add PR for running</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNama">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>2</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2Date">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date(YYMMDD):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3Weight">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>86</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4Reps">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reps:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5Distance">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Distance(km):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6Time">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Time(minutes):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTime">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2Reps">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4Weight">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5Date">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6Name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7Distance">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>90</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>321</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPRs">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>All PR's:</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>91</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Save Weight</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Save Running</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ButtonAddPR</tabstop>
  <tabstop>ButtonShowPR</tabstop>
  <tabstop>ButtonAddPRweight</tabstop>
  <tabstop>ButtonAddPRrunning</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>lineEdit_6Name</tabstop>
  <tabstop>lineEdit_5Date</tabstop>
  <tabstop>lineEdit_4Weight</tabstop>
  <tabstop>lineEdit_7Distance</tabstop>
  <tabstop>lineEdit_2Reps</tabstop>
  <tabstop>lineEditTime</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonAddPR</sender>
   <signal>clicked()</signal>
   <receiver>ButtonAddPRweight</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonAddPR</sender>
   <signal>clicked()</signal>
   <receiver>ButtonAddPRrunning</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
